variables.tf

```
@@ -216,6 +216,12 @@ variable "identity_ids" {
   default     = []
 }
 
+variable "ftps_state" {
+  description = "State of FTP/FTPS service for this Windows Function App."
```

Phrasing and allowed values (according to [variable descriptions best practices](https://equinor.github.io/terraform-baseline/best-practices/variables-and-outputs/#variable-and-output-descriptions)):

```suggestion
  description = "The state of the FTP / FTPS service for this Function App. Value must be \"AllAllowed\", \"FtpsOnly\" or \"Disabled\"."
```